### PR TITLE
feat: ai fix feedback [IDE-634]

### PR DESCRIPTION
**Pull request by acke**
### Description

Track the behavior of the users regarding the Fix AI.

The events to track:

users click button “Apply fix“
users click thumbs up or down

### Checklist

- [x] Tests added and all succeed
- [x] Linted
- [x] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Snyk Security Changelog
 
+
+## [2.11.0]
+### Changed
+- added ai fix feedback support
+
+
```
Comment (bastiandoetsch) [inline CHANGELOG.md:8]: 2.10.0 is the next release, so it should go into that.

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt [hunk 2]
```diff
@@ -299,11 +299,13 @@ class SuggestionDescriptionPanelFromLS(
                 if (!fixes.length) return;
 
                 const currentFix = fixes[diffSelectedIndex];
+                const fixId = currentFix.fixId;
+                console.log('currentFix', currentFix);
```
Comment (bastiandoetsch) [inline src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt:303]: we don't want console.logs :)

File: src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt [hunk 3]
```diff
@@ -548,6 +550,20 @@ class LanguageServerWrapper(
...
+    fun submitAutofixFeedbackCommand(fixId: String, feedback: String) {
+        if (!ensureLanguageServerInitialized()) return
+
+        try {
+            val param = ExecuteCommandParams()
+            param.command = COMMAND_CODE_SUBMIT_FIX_FEEDBACK
+            param.arguments = listOf(fixId, feedback)
+            languageServer.workspaceService.executeCommand(param)
+        } catch (ignored: Exception) {
+            // do nothing to not break UX for analytics
```
Comment (bastiandoetsch) [inline src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt:562]: should we log an exception here?

**Review by bastiandoetsch — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (acke) [reply CHANGELOG.md:8]: Fixed!

File: src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt (hunk 2 above)
Comment (acke) [reply src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt:303]: Fixed!

File: src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt (hunk 3 above)
Comment (acke) [reply src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt:562]: Fixed!

**Pull request merged**